dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/VoltageDiffNormalizer.mo

```diff
@@ -0,0 +1,53 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2015-2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+block VoltageDiffNormalizer "Block to normalize a difference between voltage measure and target level"
+
+  import Modelica;
+  import Dynawo;
+  import Modelica.Blocks.Interfaces;
+  import Modelica.Blocks.Icons.Block;
+  import Modelica.Constants;
+  import Dynawo.Types;
+
+  extends Block;
+
+  parameter Types.PerUnit gain;
+  parameter Types.PerUnit Utarget;
```

Comment:
Comment + name Target instead of UTarget so that we are not limited to voltages.